### PR TITLE
Feature/account settings functionality

### PR DESCRIPTION
**Pull request by chris7neves**
account settings page was implemented before record->update() was, so now I just added the update functionality to the page. Seems to now successfully change the user's data in the DB.

### CODE REVIEW

File: public/account_settings.php [hunk 1]
```diff
@@ -60,4 +55,22 @@ function compare_passwords($p1, $p2)
 
 <?php } ?>
 
+<?php
+if (isset($_POST['submit'])) {
+
+  empty($_POST["first_name"]) ?: $user->first_name = $_POST["first_name"];
+  empty($_POST["last_name"]) ?: $user->last_name = $_POST["last_name"];
+  empty($_POST["student_id"]) ?: $user->student_id = $_POST["student_id"];
+  empty($_POST["email"]) ?: $user->email = $_POST["email"];
```
Comment (jamestjw) [inline public/account_settings.php:64]: 😂 man this is so not DRY

Comment (jamestjw) [reply public/account_settings.php:64]: btw this form update is going to fail silently if anything goes wrong, e.g. if we aren't connected to the database, or if the student iD is not an int

Comment (chris7neves) [reply public/account_settings.php:64]: thats fair, ill add the type checking where it should be

Comment (chris7neves) [reply public/account_settings.php]: Ok I think the student id validation should be good, made a few more changes, and a database error shouldnt be silent anymore, let me know if i missed something else

**Review by jamestjw — APPROVED** [review]

**Pull request merged**